classes/task/ProcessQueueJobs.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @file classes/task/ProcessQueueJobs.inc.php
+ *
+ * Copyright (c) 2022 Simon Fraser University
+ * Copyright (c) 2022 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class ProcessQueueJobs
+ * @ingroup tasks
+ *
+ * @brief Class to process queue jobs via the schedular task
+ */
+
+namespace PKP\task;
+
+use APP\core\Application;
+use PKP\scheduledTask\ScheduledTask;
+
+class ProcessQueueJobs extends ScheduledTask
+{
+    /**
+     * @copydoc ScheduledTask::getName()
+     */
+    public function getName()
+    {
+        return __('admin.scheduledTask.processQueueJobs');
+    }
+
+
+    /**
+     * @copydoc ScheduledTask::executeActions()
+     */
+    public function executeActions()
+    {
+        if ( Application::isUnderMaintenance() ) {
+            return true;
+        }
+
+        $jobQueue = app('pkpJobQueue');
+
+        $jobBuilder = $jobQueue->getJobModelBuilder();
+
+        if ($jobBuilder->count() <= 0) {
+            return true;
+        }
+
+        while ($jobBuilder->count()) {
```

Comment:
@touhidurabir, I think we agreed you were going to add a maximum job limit for the sake of avoiding restrictive execution time limits when relying on the Acron plugin. I'll hold off the merge until your say-so on that part, thanks!

Reply:
@asmecher improvement done . check it changes at https://github.com/touhidurabir/pkp-lib/commit/c6591eb61fee87909737b464e773ee4b5bc4e178